### PR TITLE
xxsds-sdsl-lite: add version 3.0.0

### PR DESCRIPTION
**Pull request by wolfee001**
**xxsds-sdsl-lite/3.0.0**

add new version

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by SSE4 — APPROVED** [review]

**Comment by wolfee001** [comment]
someone, please?

**Review by uilianries — DISMISSED** [review]

File: recipes/xxsds-sdsl-lite/all/conanfile.py [hunk 1]
```diff
@@ -24,8 +24,11 @@ def validate(self):
     def source(self):
         source = self.conan_data["sources"][self.version]
         tools.get(**source)
```
Comment (SpaceIm) [inline recipes/xxsds-sdsl-lite/all/conanfile.py:26]:
```suggestion
        tools.get(**self.conan_data["sources"][self.version],
                  destination=self._source_subfolder, strip_root=True)
```

File: recipes/xxsds-sdsl-lite/all/conanfile.py [hunk 2]
```diff
@@ -24,8 +24,11 @@ def validate(self):
...
-        extracted_folder = "sdsl-lite-" + \
-            os.path.splitext(os.path.basename(source["url"]))[0]
+        extracted_folder = "sdsl-lite-"
+        if self.version.startswith("cci"):
+            extracted_folder += os.path.splitext(os.path.basename(source["url"]))[0]
+        else:
+            extracted_folder += self.version
```
Comment (SpaceIm) [inline recipes/xxsds-sdsl-lite/all/conanfile.py:31]:
```suggestion
```

**Review by SpaceIm — CHANGES_REQUESTED** [review]
simplify

File: recipes/xxsds-sdsl-lite/all/conanfile.py (hunk 1 above)
Comment (wolfee001) [reply recipes/xxsds-sdsl-lite/all/conanfile.py]: done

File: recipes/xxsds-sdsl-lite/all/conanfile.py (hunk 2 above)
Comment (wolfee001) [reply recipes/xxsds-sdsl-lite/all/conanfile.py]: done

File: recipes/xxsds-sdsl-lite/all/conanfile.py [hunk 3]
```diff
@@ -22,11 +22,9 @@ def validate(self):
             tools.check_min_cppstd(self, 11)
 
     def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
         source = self.conan_data["sources"][self.version]
```
Comment (uilianries) [inline recipes/xxsds-sdsl-lite/all/conanfile.py:27]:
```suggestion
```

not used

**Comment by SpaceIm** [comment]
add `required_conan_version = ">=1.33.0"` please

**Review by uilianries — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Review by SpaceIm — APPROVED** [review]

**Pull request merged**